Improved comments in ubiquiti er-x

felbinger opened the pull request:
closes #1173

danner26 commented on device-types/Ubiquiti/ER-X.yaml:
```diff
@@ -4,7 +4,7 @@ model: ER-X
 u_height: 1
 is_full_depth: false
-comments: '[EdgeRouter X - Advanced Gigabit Ethernet Router](https://www.ui.com/edgemax/edgeswitch-10x/)'
+comments: '[EdgeRouter X - Advanced Gigabit Ethernet Router](https://store.ui.com/collections/operator-edgemax-routers/products/edgerouter-x)'
```
I actually think this should be the datasheet: https://dl.ui.com/datasheets/edgemax/EdgeRouter_X_DS.pdf

felbinger replied:
Adjusted

danner26 commented:
Hello, unfortunately today we had to make a change to the schema in which weight is defined by. The upstream NetBox API requires weight to be passed to it in the following format:
```
weight: 12.01
weight_unit: lb
```

Due to this change, I have updated your branch against the current master. This might require you to update your PR, if you had weight currently defined, so that your PR is in-line with the new weight schema. If you have any issues, please let me know or refer to [this PR](https://github.com/netbox-community/devicetype-library/pull/1187) for more information.

felbinger commented:
> Hello, unfortunately today we had to make a change to the schema in which weight is defined by. The upstream NetBox API requires weight to be passed to it in the following format:
> 
> ```
> weight: 12.01
> weight_unit: lb
> ```
> 
> Due to this change, I have updated your branch against the current master. This might require you to update your PR, if you had weight currently defined, so that your PR is in-line with the new weight schema. If you have any issues, please let me know or refer to [this PR](https://github.com/netbox-community/devicetype-library/pull/1187) for more information.

Updated, I also added the weight in the given format. I may be a good idea to squash this pr with the title of the first commit.
